x/evm/types/params_test.go

```diff
@@ -47,9 +48,14 @@ func TestParamsValidate(t *testing.T) {
 		},
 		{
 			"invalid chain config",
-			NewParams("ara", true, true, ChainConfig{}, 2929, 1884, 1344),
+			NewParams("ara", true, true, ChainConfig{}, sdk.NewDecWithPrec(50, 2), 2929, 1884, 1344),
 			false,
```

Comment:
Why is `expError = false` for an invalid chain config?

Reply:
This test is useless, just removed it. Good catch!